openshift/templates/dancer-mysql.json

```diff
@@ -115,10 +126,7 @@
               "secret": "${GITHUB_WEBHOOK_SECRET}"
             }
           }
-        ],
-        "postCommit": {
-          "script": "perl -I extlib/lib/perl5 -I lib t/*"
-        }
```

Comment:
For other repos, the origin copies seemed to be newer (e.g. the inclusion of the "message" field in the template), so I used that as a heuristic for all updates. Looks like it was wrong for this file. I will restore it.